### DISCUSSION TITLE
How can I design my own entities, entity types, color to visualize the text in spacy without nlp just mark?

### DISCUSSION THREAD

**Discussion by YijianLiu** [explosion/spaCy#7578] [Help: Coding & Implementations] [answered]
Thanks to your so nice visualize, but now I just want mark the text only but no use nlp, for example,
I give the sentence: Mike likes dog, and then I input data:
{ent: 'like', type: 'PERSON', color:'#D2423'} 
{ent: 'dog, type: 'LOC', color:'#E3213'} 
Then it will mark like as PERSON and dog as LOC,
I really want to use your nice mark functino without doing any other nlp,
many thanks~ HELPHELPHELP!

**Comment by polm** [comment, marked as answer]
The simplest way to do this would be to use `nlp.make_doc`, which won't run any of the pipelines, and then manually set the annotations. In order to manually set the annotations you'll need to get character offsets. You might find [Example.from_dict](https://spacy.io/api/example#from_dict) helpful.
